src/OrbitCore/Path.h

```diff
@@ -14,5 +14,6 @@ namespace Path {
 [[nodiscard]] std::filesystem::path CreateOrGetCaptureDir();
 [[nodiscard]] std::filesystem::path CreateOrGetDumpDir();
 [[nodiscard]] std::filesystem::path CreateOrGetOrbitAppDataDir();
-[[nodiscard]] std::filesystem::path GetLogFilePathAndCreateDir();
+[[nodiscard]] std::filesystem::path CreateOrGetLogDir();
```

Comment:
we probably want to test this function as well.

Reply:
Updated the test file. Thanks!